ha_cluster_exporter.go

```diff
@@ -163,7 +165,7 @@ func resetClusterMetrics() {
 	err = prometheus.Register(clusterNodes)
 	if err != nil {
 		log.Println("[ERROR]: failed to register clusterNode metric. Perhaps another exporter is already running?")
-		log.Panic(err)
+		log.Println(err)
 	}
```

Comment:
maybe we should return this error instead, and use the logger in the caller (`main()`)?